### PR TITLE
feat(BitBucket): add BitBucket server docs

### PR DESCRIPTION
**Pull request by nabeelsaabna**

### CODE REVIEW

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/configure-renovate-bitbucket-server.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+# Stage 2: Install Renovate CE/EE Application Server
+
+## Configure the Docker files / Helm charts
+Fetch the example docker-compose file or Helm chart configuration files and edit accordingly.
+Example files available here:
+- Docker files (Renovate CE / Renovate EE)
+- Helm charts (Renovate CE / Renovate EE)
+
+Edit the docker files / helm chart values to provide the required environment variables.
+Refer to [LINK] for a full list of Renovate CE/EE server variables.
```
[the remark this answers is not in the document]
Comment (nabeelsaabna) [reply docs/configure-renovate-bitbucket-server.md:103]: We don't this file yet!

File: docs/configure-renovate-bitbucket-server.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+`MEND_RNV_ADMIN_API_ENABLED` - APIs are off by default. Set this to true to enable admin APIs.
+Needs `MEND_RNV_SERVER_API_SECRET` to be set.
+`MEND_RNV_SQLITE_FILE_PATH` - Mount the DB file to disc. Ensure volume mount is configured below.
+`MEND_RNV_LOG_HISTORY_DIR` - Mount the Renovate job logs (Note: This is the Job logs; Not the server/worker machine logs.)
+`RENOVATE_REPOSITORY_CACHE` - Faster performance running Renovate on repos on subsequent scans.
+
+## Run the Server
+If using Docker, run the Docker Compose file after all values have been correctly inserted.
+
+> docker-compose -f docker-compose-bitbucket.yaml up  [Check this]
```
[the remark this answers is not in the document]
Comment (nabeelsaabna) [reply docs/configure-renovate-bitbucket-server.md:126]: Not sure what should this be :-/

File: docs/configure-renovate-bitbucket-server.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+
+**Force App Sync via API call**
+
+To trigger the sync immediately, call the sync API (/api/sync) using a tool like Postman.
+Requires
+
+Note: To run APIs, ensure Renovate Server has EnvVar `MEND_RNV_ADMIN_API_ENABLED=true`
+
+[POST]   http://<RENOVATE-SERVER-URL>/api/sync
+Authorization: <MEND_RNV_SERVER_API_SECRET>
```
[the remark this answers is not in the document]
Comment (nabeelsaabna) [reply docs/configure-renovate-bitbucket-server.md:209]: fixed by separate commit

File: docs/configure-renovate-bitbucket-server.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nabeelsaabna) [reply docs/configure-renovate-bitbucket-server.md:126]: will need to add a new file, prefer to keep it for a another PR

File: docs/configure-renovate-bitbucket-server.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nabeelsaabna) [reply docs/configure-renovate-bitbucket-server.md:103]: will need to add a new file, prefer to keep it for a another PR

File: docs/configure-renovate-bitbucket-server.md [hunk 4]
```diff
@@ -0,0 +1,455 @@
...
+Accepts a 5-part cron schedule. Defaults to `0 */4* * * *` (every 4 hours, on the hour).
+
+**Force App Sync via API call**
+
+To trigger the sync immediately, call the sync API (/api/sync) using a tool like Postman.
+Requires
+
+Note: To run APIs, ensure Renovate Server has EnvVar `MEND_RNV_ADMIN_API_ENABLED=true`
+
+[POST]   http://<RENOVATE-SERVER-URL>/api/sync
```
Comment (Gabriel-Ladzaretti) [inline docs/configure-renovate-bitbucket-server.md:310]: .

**Review by OBITOONDEADO1MZ — APPROVED** [review]

**Pull request merged**